File: source/zombie/debug/options/IDebugOption.java
// Decompiled on Sat Jan 17 08:24:00 EST 2026 with Zomboid Decompiler v0.2.3 using Vineflower.
package zombie.debug.options;

public interface IDebugOption {
    String getName();

    IDebugOptionGroup getParent();

    void setParent(IDebugOptionGroup parent);
}

File: PZ-Switch/src/zombie/debug/options/IDebugOption.cpp
#include "zombie/debug/options/IDebugOption.h"

namespace zombie {
namespace debug {
namespace options {

std::string IDebugOption::getName() {
 // TODO: Implement getName
 return "";
}

IDebugOptionGroup IDebugOption::getParent() {
 // TODO: Implement getParent
 return nullptr;
}

void IDebugOption::setParent(IDebugOptionGroup parent) {
 // TODO: Implement setParent
}

} // namespace options
} // namespace debug
} // namespace zombie
